lib/metadata/linux/MiqRpmPackages/Sqlite.rb

```diff
@@ -0,0 +1,55 @@
+class MiqRpmPackages
+  class Sqlite < MiqRpmPackages
+    def initialize(fs, dbFile)
+      @fs           = fs
+      @db_file_path = dbFile
+
+      @rpmdb_tempdir = nil
+      @rpmdb_path    = nil
+
+      if fs.present?
+        @rpmdb_tempdir = Dir.mktmpdir("rpmdb-")
+        FileUtils.mkdir_p("#{@rpmdb_tempdir}/var/lib/rpm")
```

Comment:
For Mac support, the path needs to be `"#{@rpmdb_tempdir}/opt/homebrew/var/lib/rpm"`